src/java/generator/MapGenerator.java

```diff
@@ -32,8 +32,10 @@
 	public static final String VERSION = "0.1.6";
 
 	//read from cli args
+//	private static Random randomseed = new Random();
 	private static String FOLDER_PATH = ".";
 	private static String MAP_NAME = "debugMap";
+//	private static long SEED = randomseed.nextLong();
```

Comment:
Let's not leave dead code around... just delete it

Reply:
Will do